src/PointEval/point_values.jl

```diff
@@ -24,7 +24,7 @@ struct PointValues{CV} <: AbstractValues
     PointValues{CV}(cv::CV) where {CV} = new{CV}(cv)
 end
 
-PointValues(cv::CellValues) = PointValues(eltype(cv.M), cv.ip, cv.gip)
+PointValues(cv::CellValues) = PointValues(eltype(shape_value(cv,1,1)), cv.fun_values.ip, cv.geo_values.ip)
```

Comment:
```suggestion
PointValues(cv::CellValues) = PointValues(shape_value_type(cv), get_function_interpolation(cv), get_geometry_interpolation(cv))
```

Reply:
Nice catch - this seems wrong, would only work for scalar function interpolation 
It has to be the geometric shape because this is always scalar.

Reply:
Just wanted to ask again here :P

Reply:
Aha, would actually work when I think about it (because `eltype(1.0)=eltype(Vec((1.0,))=Float64`), but it is quite hard to read. Better to just take the type from the `detJdV`!